cmd/prometheus/config.go

```diff
@@ -242,6 +242,13 @@ func init() {
 
 func parse(args []string) error {
 	err := cfg.fs.Parse(args)
+
+	unkown_args := cfg.fs.Args()
+
+	if len(unkown_args) != 0 {
```

Comment:
since unknown_args isn't used anywhere else, you could make the scope smaller:
```
if unknown := cfg.fs.Args(); len(unknown) != 0 {
    ...
}
```